store/sqlstore/group_store.go

```diff
@@ -909,6 +909,41 @@ func (s *SqlGroupStore) groupsBySyncableBaseQuery(st model.GroupSyncableType, t
 			OrderBy("ug.DisplayName")
 	}
 
+	if opts.FilterAllowReference && t == selectGroups {
+		query = query.Where("ug.AllowReference = 1")
```

Comment:
Thanks, I had the same issue when I ran the Postgres store tests locally, just didn't get to lookup the existing pattern and also wanted to see if they also fail in Jenkins/CI (sometimes I had surprises when running tests locally and on the PR), I will try your suggestion above "(Eq{"ug.AllowReference": true})", I suspect it will fix it.

Reply:
`ug.AllowReference = true` looks to work ok